testing/README.md

```diff
@@ -649,9 +644,20 @@ check doesn't have to match the event being generated: by the time that Juju
 sends a pebble-check-failed event the check might have started passing again.
 
 ```python
+from ops import pebble
+
 ctx = scenario.Context(MyCharm, meta={"name": "foo", "containers": {"my_container": {}}})
-check_info = scenario.CheckInfo("http-check", failures=7, status=ops.pebble.CheckStatus.DOWN)
-container = scenario.Container("my_container", check_infos={check_info})
+layer = pebble.Layer({
+    "checks": {"http-check": {"override": "replace", "startup": "enabled", "failures": 7}},
+})
+check_info = scenario.CheckInfo(
+    "http-check",
+    status=pebble.CheckStatus.UP,
+    level=layer.checks["http-check"].level,
+    startup=layer.checks["http-check"].startup,
+    threshold=layer.checks["http-check"].threshold,
+)
+container = scenario.Container("my_container", check_infos={check_info}, layers={"layer1": layer})
```

Comment:
Thanks - I missed this in the recent consistency check PR. A good argument for having this environment run automatically (and a better one for removing these examples from the README).